### DISCUSSION TITLE
How to make GraphQL requests via AppSync or Amplify from this cpp sdk?

### DISCUSSION THREAD

**Discussion by grimaldini** [aws/aws-sdk-cpp#2110] [General]
I've seen documentation for how to make a GraphQL request using Amplify for languages like JS but not for C++? Is this not currently supported? I noticed there are some classes GraphQLApi, and there's a AppSync client even but I'm not sure if that serves the same purpose.

**Comment by jmklix** [comment]
You can check the api docs [here](https://sdk.amazonaws.com/cpp/api/LATEST/namespace_aws_1_1_app_sync.html) for how to make GraphQL requests via AppSync. It doesn't look like it is currently supported via Amplify.
Are you looking for [examples](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/cpp), because the requests should look similar the the JS ones.

**Reply by grimaldini** [reply]
I have already checked those docs.
The first one is just a doc gen from the comments in the code, which is not very helpful on how to use it.
The second one does not include any AppSync examples...

**Reply by jmklix** [reply]
This isn't the best solution, but you can look at examples in other languages to see how to setup the requests and then use that to do the same with the c++ sdk
